cyences_app_for_splunk/default/macros.conf

```diff
@@ -239,12 +243,15 @@ definition = cyencesdevicemanager operation="getdevices" \
 | eval kaspersky_status=kaspersky_status \
 | eval tmp_kaspersky_status = kaspersky_status, kaspersky=case(tmp_kaspersky_status == "Present", 0, tmp_kaspersky_status == "Disabled", 1) \
 | fieldformat kaspersky=tmp_kaspersky_status \
+| eval splunk_status=case(isnull(splunk_last_event), "-", 1==1, splunk_forwarder_type) \
```

Comment:
splunk_status word is misleading as one would expect to have up/down status.

Instead we can say splunk_type maybe.